phone/src/locale/pl.json

```
@@ -0,0 +1,289 @@
+        "INVALID_PHONE_NUMBER": "Nieprawidłowy numer telefonu",
+        "MESSAGE_CONVERSATION_DUPLICATE": "Ta rozmowa już trwa!",
+        "MESSAGE_GROUP_CREATE_FAILED": "Nie udało się utworzyc rozmowy",
+        "MESSAGE_GROUP_CREATE_MINE": "Nie możesz dodać sam siebie!",
+        "FETCHED_MESSAGES_FAILED": "Nie udało się pobrać wiadomości",
+        "DELETE_MESSAGE": "Usuń wiadomość",
+        "DELETE_MESSAGE_FAILED": "Nie udało się usunąć wiadomości",
+        "DELETE_CONVERSATION_FAILED": "Nie udało się usunąć konwersacji"
+      },
+      "SEARCH_PLACEHOLDER": "Szukaj wiadomości...",
+      "DELETE_CONVERSATION": "Usuń konwersację",
+      "NEW_MESSAGE": "Wiadomość...",
+      "NEW_MESSAGE_GROUP": "Numer telefonu...",
+      "GROUP_CHAT_LABEL": "Nazwa czatu grupowego...",
+      "MEDIA_OPTION": "Aparat / Galeria",
+      "NEW_MESSAGE_GROUP_SUBMIT": "Wyślij",
+      "INPUT_NAME_OR_NUMBER": "Wyszukaj kontakty lub wpisz numer i naciśnij enter",
+      "ACTIONS_TITLE": "Dzialania",
+      "ME": "Ja"
+    },
+    "RELATIVE_TIME": {
+      "future": "w %s",
+      "past": "%s temu",
+      "s": "kilka sekund",
+      "m": "minuta",
+      "mm": "%d minut",
+      "h": "godzina",
+      "hh": "%d godzin",
+      "d": "dzien",
+      "dd": "%d dni",
+      "M": "miesiąc",
+      "MM": "%d miesięcy",
+      "y": "rok",
+      "yy": "%d lat"
+    },
+    "GENERIC": {
+      "LOADING": "Ładowanie...",
+      "WAIT": "Proszę czekać...",
+      "SAVE": "Zapisz",
+      "EDIT": "Edytuj",
+      "UPDATE": "Aktualizacja",
+      "REQUIRED": "Wymagany",
+      "DELETE": "Usunąć",
+      "REPORT": "Zgłoszenie",
+      "CALL": "Połączenie",
+      "MESSAGE": "Wiadomość",
+      "CLOSE": "Zamknąć",
+      "CANCEL": "Anuluj",
+      "TITLE": "Tytuł",
+      "CONTENT": "Zawartość",
+      "WRITE_TO_CLIPBOARD_TOOLTIP": "Skopiuj {{ content }}",
+      "WRITE_TO_CLIPBOARD_MESSAGE": "Skopiowano {{content}} do schowka!"
+    },
+    "COMING_SOON": "Wkrótce...",
+    "INITIALIZING": "Inicjowanie",
+    "APPS_DIALER": "Telefon",
+    "APPS_CONTACTS": "Kontakty",
+    "APPS_MESSAGES": "Wiadomości",
+    "APPS_CALCULATOR": "Kalkulator",
+    "APPS_SETTINGS": "Ustawienia",
+    "APPS_BANK": "Bankowość WhoDis",
+    "APPS_NOTES": "Notatnik",
+    "APPS_EXAMPLE": "Przykładowa aplikacja",
+    "APPS_TWITTER": "Twitter",
+    "APPS_MARKETPLACE": "Rynek",
+    "APPS_CAMERA": "Kamera",
+    "APPS_MATCH": "Matchmaker",
+    "APPS_CALCULATOR_COPIED": "Wynik został skopiowany do schowka!",
+    "GENERIC_CLICK_TO_REVEAL": "Kliknij, aby odsłonić",
+    "GENERIC_LOADING": "Ładowanie...",
+    "GENERIC_WAIT": "Proszę czekać...",
+    "GENERIC_SAVE": "Zapisz",
+    "GENERIC_EDIT": "Edytuj",
+    "GENERIC_UPDATE": "Aktualizacja",
+    "GENERIC_REQUIRED": "Wymagany",
+    "GENERIC_DELETE": "Usunać",
+    "GENERIC_REPORT": "Zgłoszenie",
+    "GENERIC_CALL": "Połączenie",
+    "GENERIC_MESSAGE": "Wiadomość",
+    "GENERIC_CLOSE": "Zamknij,
```

```suggestion
    "GENERIC_CLOSE": "Zamknij",
```
Missing quotation here it looks like